command/cat.go

```diff
@@ -66,12 +68,13 @@ var catCommand = &cli.Command{
 }
 
 // Cat prints content of given source to standard output.
-func Cat(ctx context.Context, src *url.URL) error {
+func Cat(ctx context.Context, src *url.URL) (err error) {
+	defer stat.Collect("Cat", time.Now(), &err)()
```

Comment:
Instead of adding stat logic into each command, why we cannot collect stats in `app.ExitErrHandler()`? Smth like:

```go
ExitErrHandler: func(c *cli.Context, err error) {
        stat.Collect(c.Command.FullName(), err)
},
```

Also, I personally found it unnecessary to calculate elapsed time for each command execution. Users may access this information through :

```
time s5cmd ...
```

I think stat mode is useful for showing the number of successful/failed operations, especially in batch mode.